framework/components/ecl-fields/ecl-fields.config.js

```diff
@@ -1,17 +1,33 @@
 module.exports = {
   title: 'Fields',
   label: 'Fields',
-  status: 'planned',
-  collated: false,
+  status: 'ready',
+  collated: true,
+  collator(markup, item) {
+    return `
+      <!-- Start: @${item.handle} -->\n
+      <h3 class="ecl-heading ecl-heading--h3">${item.label}\n</h3>\n
+      ${markup}\n
+      <!-- End: @${item.handle} -->\n
+    `;
+  },
   tags: ['atom'],
   variants: [
     {
       name: 'default',
       label: 'Default',
+      context: {
+        label: 'Name',
+        body: 'John Doe',
```

Comment:
Well here we have an issue.
With our current implementation, we don't have multiple examples of each components. Maybe we could create some example pages, but the component itself should remain as simple as possible (for composition)